### ISSUE TITLE
Ability to Force Imports

### ISSUE THREAD

**Issue by Triphammer** [satisfactory-factories/application#314]
Suppose I already have a supply line created to generate 100 Alclad sheets a minute in my world.

If I want to create a new factory that uses Alclad sheets in this app, I have to fake up a factory that has the entire process from raw bauxite right through Alclad sheets, and deals with the extract products and waste and so on, just to add them as an import.

It would be VERY helpful to have a "External Import Source" or something like that, where I could just say "I have 100 Alclad Sheets a minute that I have from an outside source, just trust me and treat this as a source, and only notify me if this is insufficient to meet my factory needs".

If the input isn't enough, then treat it just like a normal source where it would force me to up production by increasing the number that I provide, but how that's done is my responsibility outside of this plan.

**Comment by Maelstromeous** [comment]
Yeah I've been thinking of allowing this, but special consideration has to be accounted here. A large amount of the planner's calculations is checking for import validity, and it does special calculations plan wide to ensure that the plan is accurate.

I like the concept, I also have been considering removing the inate "Raw Resources assumed correct" and giving the user the option to either mark it as "don't worry about this input" and not require the extra layer of mine "factories", or giving the user the choice whether they want this level of complexity.

I'll accept this issue and consider implementation, no promises as to when though as it's major change in how the planner works and does math.

**Comment by Triphammer** [comment]
I had no idea it would be as substantial a change as that.  Typical user, right?  :)

Thanks for the consideration.  Keep up the awesome work.

**Comment by Maelstromeous** [comment]
We're not assuming anything at all now. Rejected as part of #503
